client/app/components/packages/rwcds-form/proposed-project-actions-editor.hbs

```diff
@@ -1,39 +1,62 @@
+        {{#if (lookup-action-type form.saveableChanges.dcpZoningresolutiontype)}}
+          <LabsUi::IconTooltip
+            @tip={{lookup-action-type form.saveableChanges.dcpZoningresolutiontype}}
+            @icon="info-circle"
+            @side="bottom"
+            @transform="shrink-4 up-2"
+            @fixedWidth={{true}}
+          />
+        {{/if}}
+      </span>
+      {{#if this.zrSectionNumber}}
+        <span data-test-zrsectionnumber>{{this.zrSectionNumber}}</span>
       {{/if}}
-    </span>
-    {{#if this.zrSectionNumber}}
-      <span data-test-zrsectionnumber>{{this.zrSectionNumber}}</span>
-    {{/if}}
-    {{#if this.hasModifiedZrSectionNumberQuestion}}
-      <Q.Prompt>
-        Which sections of the Zoning Resolution does this modify?
-      </Q.Prompt>
-      <small class="help-text display-block">
-        Provide the Zoning Resolution section number(s). Ex. ZR Sec. 42-10 and 43-17
-      </small>
-      <Form.Field
-        @attribute="dcpModifiedzrsectionnumber"
-        as |TextInput|
-      >
-        <TextInput
-          type="text"
-          maxlength="250"
-          data-test-input="dcpModifiedzrsectionnumber"
-        />
-      </Form.Field>
-    {{else if this.zrAppendixF}}
-      This action {{if @rwcdsForm.saveableChanges.dcpIncludezoningtextamendment "includes" "does not include"}} MIH. If this is a mistake, please contact City Planning. 
+    </Q.Prompt>
+    {{#if (or this.hasModifiedZrSectionNumberQuestion this.hasZrSectionTitleQuestion)}}
+      {{#if this.hasModifiedZrSectionNumberQuestion}}
+        <Q.Prompt>
+          Which sections of the Zoning Resolution does this modify?
+        </Q.Prompt>
+        <small class="help-text display-block">
+          Provide the Zoning Resolution section number(s). Ex. ZR Sec. 42-10 and 43-17
+        </small>
+        <form.Field
+          @attribute="dcpModifiedzrsectionnumber"
+          as |TextInput|
+        >
+          <TextInput
+            type="text"
+            maxlength="25"
+            data-test-input="dcpModifiedzrsectionnumber"
+          />
+        </form.Field>
+      {{/if}}
+      {{#if this.hasZrSectionTitleQuestion}}
+        <Q.Prompt>
+          What is the title for the section of the Zoning Resolution that this action is pursuant to?
+        </Q.Prompt>
+        <small class="help-text display-block">
+          Example: “Landmark preservation in all districts”. Refer to the Zoning Resolution to find the title for the section number.
+        </small>
+        <form.Field
+          @attribute="dcpZrsectiontitle"
+          as |TextInput|
+        >
+          <TextInput
+            type="text"
+            maxlength="100"
+            data-test-input="dcpZrsectiontitle"
+          />
+        </form.Field>
+      {{/if}}
+      {{#if this.zrAppendixF}}
+        This action {{if @rwcdsform.saveableChanges.dcpIncludezoningtextamendment "includes" "does not include"}} MIH. If this is a mistake, please contact City Planning.
```

Comment:
Sorry, I should have caught this last time. This here should be just `form.saveableChanges. dcpIncludezoningtextamendment`, right? Not `@rwcdsform.saveableChanges.dcpIncludezoningtextamendment`

Reply:
Otherwise, this PR looks great!! Thanks Taylor!

Reply:
So we need two models in this component. 1) affected-zoning-resolution and 2) rwcds-form. 

This attribute does not come from affected-zoning-resolution but instead is an outlier and comes from rwcds-form so I passed in rwcds-form as well.

affected-zoning-resolution has a MANY to ONE relationship with rwcdsForm. But based on how the component is set up (with validations on saveableForm and passing in one affected-zoning-resolution at a time), it didn't make sense to me to pass in rwcdsForm.affectedZoningResolutions. Since it's just that one little display (and not based on anything being patched or changed) I figured I'd just pass in rwcdsForm as well.

Let me know if you disagree with this approach. I tried to implement the one that felt easiest to understand.

Reply:
Ohhh. Sorry my bad, thank you for explaining. This is an interesting caveat with our nested form design. If we want a component to interact with multiple forms, we have to pass each one into that component separately.

Reply:
I think we should leave it this way for this PR for now, since it follows the current pattern. Though we should definitely think about how we can make this workflow a bit nicer. 

I did just notice how the forms are passed into the editor:
**proposed-project-actions.hbs**
```
          <Packages::RwcdsForm::ProposedProjectActionsEditor
            @form={{affectedZoningResolutionForm}}
            @rwcdsForm={{@form}}
          />
```

I don't know what's best.... Just thinking out loud but perhaps
- If we do decide to pass each form in as a separate parameter, it would be cleaner to make the parameter names consistently expressive. `@form`  should probably be something like `@zrForm`
- Or perhaps we can pass in a hash...
```
          <Packages::RwcdsForm::ProposedProjectActionsEditor
            @forms={{hash
                 zForm=affectedZoningResolutionForm
                 rwcdsForm=@form
             }}
          />
```

Reply:
Okay that's a good point! I'll update the form names